eng/Build.props

```diff
@@ -191,6 +191,7 @@
                           $(RepoRoot)src\Security\**\src\*.csproj;
                           $(RepoRoot)src\SiteExtensions\**\src\*.csproj;
                           $(RepoRoot)src\Tools\**\src\*.csproj;
+                          $(RepoRoot)src\Logging\**\src\*.csproj;
```

Comment:
Add this about https://github.com/dotnet/aspnetcore/pull/20421/files#diff-991506ab3bfb2ea4767ed342c4f95726R154 too

Reply:
I don't understand what do you mean? The link is to latest package references?

Reply:
These item group includes are conditional and logging should be on both sides of the condition.

If you prefer, it's possible to remove duplication between this and the above items on the other side of the `$(BuildMainlyReferenceProviders)` check. Could for example add a property that's normally empty but contains `src\` when `$(BuildMainlyReferenceProviders)` is `true`. I didn't get around to doing that, sorry.